scripts/gen_merkle_dist.js

```diff
@@ -1,56 +1,119 @@
 // Script that generates a new Merkle Distribution and outputs the data to JSON files
 
 const fs = require("fs")
+const shell = require("shelljs")
+const dotenv = require("dotenv").config()
```

Comment:
Seems to be unused

Reply:
shelljs is used to run rewards.sh [(line](https://github.com/threshold-network/merkle-distribution/blob/62dd2828d7f19e7e56fff52155d8ff35c3607220/scripts/gen_merkle_dist.js#L65)

dotenv is neccesary to use env variables in [line](https://github.com/threshold-network/merkle-distribution/blob/62dd2828d7f19e7e56fff52155d8ff35c3607220/scripts/gen_merkle_dist.js#L35)